basic-application/basic-application-core/src/main/java/fr/openwide/core/basicapp/core/business/referential/service/ICityService.java

```diff
@@ -0,0 +1,9 @@
+package fr.openwide.core.basicapp.core.business.referential.service;
+
+import fr.openwide.core.basicapp.core.business.referential.model.City;
+import fr.openwide.core.jpa.business.generic.service.IGenericEntityService;
+
+public interface ICityService extends IGenericEntityService<Long, City> {
+
+	City getByUniqueKeyIgnoreCase(String libelle);
```

Comment:
I may be mistaken but... I think this method is never used.

I guess it was supposed to be used in a unicity validator when editing a City. I think this would be relevant to add this for every "simple" genericListItem. You would implement your unicity validator using `fr.openwide.core.jpa.more.business.generic.service.IGenericListItemService.getByFieldIgnoreCase(EntityPath<E>, StringExpression, String)`.